ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -299,20 +299,35 @@ objects:
         echo "Removing bootstrap host from control plane api pool"
         (cd /tmp/artifacts/terraform && terraform apply -auto-approve=true -var=bootstrap_dns="false")
 
-        echo "Approving any pending CSR requests"
-        oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs oc adm certificate approve
+        function approve_csrs() {
+          while true; do
+            if [[ ! -f /tmp/install-complete ]]; then
+              oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve
+              sleep 15 & wait
```

Comment:
Backgrounding slow processes gets them into the `jobs` output so we can [TERM them for graceful shutdown][1].  I don't think we care about gracefully stopping `sleep`, it won't cause any problems getting hard-killed.  Can we just make this line `sleep 15`, or am I missing something?

[1]: https://github.com/openshift/release/blob/d143934d7408a140bcb4037c29c9cf0fadfb293d/ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml#L221